plugin/src/main/scala/plugin/package.scala

```diff
@@ -0,0 +1,49 @@
+// See LICENSE for license details.
+
+import java.io.FileWriter
+
+package object plugin {
```

Comment:
Is this mainly a collection of debugging utils? This seems kind of generic, almost something I'd expect that we should be able to pull in from the Scala compiler code? (but I may also be wrong)